src/create-event-handlers/on-time.js

```diff
@@ -1,41 +1,62 @@
 function onTime(event) {
-  const { hasFired } = this.state;
-  const { position, duration } = event;
-  let hasChanged = false;
+  const { hasFired, previousPosition } = this.state;
+  const { duration, position } = event;
 
-  if (!hasFired.threeSeconds && position > 3) {
+  const durationInteger = Math.floor(duration);
+  const currentPositionInteger = Math.floor(position);
+  const previousPositionInteger = previousPosition || 0;
```

Comment:
Also, wondering if we can just short circuit this now? Seems to me that if currentPosition === previousPosition, there is never anything to be done, right?